arb_os/chainParameters.mini

```diff
@@ -52,10 +53,11 @@ public impure func chainParams_gotParamsMessage(sender: address, data: ByteArray
             baseStake: bytearray_get256(data, 3*32),
             stakingToken: address(bytearray_get256(data, 4*32)),
             owner: address(bytearray_get256(data, 5*32)),
+            blocksPerSend: bytearray_get256(data, 6*32),
```

Comment:
Is there some way we could set this to a reasonable secure default and then have an extra option to customize it?

Reply:
Fixed this.